### ISSUE TITLE
RDSendLine doesn't send newline

### ISSUE THREAD

**Issue by protist** [jalvesaq/Nvim-R#237]
I have the following minimal `~/.vimrc`:
```
filetype plugin on
nmap <Space> <Plug>RDSendLine
```
I then open the following Rmd file:
```
```{r code}
print("foo")
``` 
```
I start R with `\rf`, move the cursor over the code and press `space`. The code is sent to the R terminal, but the enter/newline isn't appended. I try to send it again, but the command doesn't run.
```
> print("foo")print("foo")print("foo")print("foo")
```
I get the same behaviour with `\rq`, i.e. the newline isn't sent to the terminal. The code window thinks R has quit, but it hasn't.

Oddly enough, I created a new user account on my system, and cannot replicate this issue with the same vimrc (and nothing in `~/.Rprofile`). I checked the startup vim scripts for each user with `:scriptnames`, and they are identical.

VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Oct  7 2017 13:50:55) Included patches: 1-1176
Nvim-R 0.9.10-1
Up-to-date Arch Linux

**Comment by jalvesaq** [comment]
@protist Did you discover what was causing the problem?

**Comment by protist** [comment]
@jalvesaq No, I didn't. However, I recently changed computers (although I copied my home directory). I just tested again on the new system, and I still see this issue.

**Comment by jalvesaq** [comment]
I'm sorry for that, but I can't help because I can't replicate the problem.

**Comment by protist** [comment]
Hmmm… I tested a few things again on this new system.

1) I created a new user account. I removed all home directory vim scripts from my main account, so that both accounts gave the same `:scriptnames` output. The new user account worked, the main failed.
2) I removed all my other 17 vim plugins anyway, but it still fails.
3) I can't think what else might be influencing my main account but not the new one. In 1 above, I tested by logging into the fresh account with `sudo su test`. In case it is something to do with the X session, I tested on a tty, but it still fails.

I'm not sure how helpful any of this is…

**Comment by jalvesaq** [comment]
There must be a difference... Terminal emulator, vim version, .tmux.conf, .bashrc, .inputrc ...

**Comment by jalvesaq** [comment]
Thanks for your feedback!
